courses/free-courses-es.md

```diff
@@ -206,6 +214,7 @@
 * [Curso gratuito de JavaScript](https://argentinaprograma.com) - Fabricio Sodano (Argentina Programa)
 * [Curso gratuito de Next.js y Firebase](https://www.youtube.com/playlist?list=PLV8x_i1fqBw1VR86y4C72xMGJ8ifjBwJ6) - Miguel Ángel Durán «midudev»
 * [Curso JSON. De Novato a Experto](https://www.youtube.com/playlist?list=PLrDTf5qnZdEAiHO19QB9hq5QXAef1h8oY) - Camilo Martínez "Equimancho"
+* [CURSO REACT NATIVE desde CERO](https://www.youtube.com/watch?v=qi87b6VcIHY&t=1004s&ab_channel=midudev) - Miguel Ángel Durán (midudev)
```

Comment:
```suggestion
* [Curso React Native desde CERO](https://www.youtube.com/watch?v=qi87b6VcIHY) - Miguel Ángel Durán (midudev)
```
Is CERO an abbreviation? We don't use ALL CAPS title except for abbreviations.

Reply:
I'm Sorry. The word CERO is not an abbreviation and a correction to the title has already been uploaded